scripts/configure-dh.sh

```diff
@@ -364,6 +364,12 @@ if [[ $LIGHTSPEED_INTEGRATION == "true" ]]; then
     fi
     echo -n "."
 fi
+# Only use for running end-to-end tests in CI, keep away from production
+if [[ $CI_DISABLE_AUTH == "true" ]]; then
```

Comment:
Thinking for this variable a declaration to follow the others that defaults to `false` and a generalized name, such as `RHDH_DISABLE_AUTH_POLICY`, as there could be other reasons to disable the authentication policy:

```suggestion
if [[ $RHDH_DISABLE_AUTH_POLICY == "true" ]]; then
```

Reply:
Renamed the variable as suggested, added `false` as default, and a snippet in the doc for explanation.